x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/classification_exploration/evaluate_panel.tsx

```diff
@@ -349,14 +374,25 @@ export const EvaluatePanel: FC<Props> = ({ jobConfig, jobStatus, searchQuery })
                           <EuiDataGrid
                             data-test-subj="mlDFAnalyticsClassificationExplorationConfusionMatrix"
                             aria-label="Classification confusion matrix"
```

Comment:
Another label we should have internationalized.

Reply:
Added in 422c7aacdfc9d129a7877af2d9f1c8ebede15f80